### PR TITLE
#184637234 fixed the seeders and migrations issue

### PR DESCRIPTION
**Pull request by masterchief-00**
#### What does this PR do?

- Created migrations and seeders

#### Description of Task to be completed?
- Configure The App to use PostgreSQL as DB Engine and Sequelize ORM
#### How should this be manually tested?

-  `npm run migrate` for migration
- `npm run seed` for seeding
- more database related commands can be found in the package.json file in scripts section

#### Any background context you want to provide?
 none
#### What are the relevant pivotal tracker/Trello stories?
 Tracker: 184637234
#### Screenshots (if appropriate)
 none
#### Questions:
 none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,18 @@
...
   "scripts": {
-    "start": "node index",
+    "start": "node ./src/index",
     "dev": "nodemon --exec babel-node ./src/index",
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint ./",
-    "lint:fix": "eslint ./ --fix"
+    "lint:fix": "eslint ./ --fix",
+    "migrate:seed": "npm run migrate && npm run seed",
+    "migrate": "npm run migrate:undo:all && npm run migrate:all",
```
Comment (fab-ryan) [inline package.json]: you can't delete all migration use **npm run migrate** only  that's enough

File: package.json [hunk 2]
```diff
@@ -4,11 +4,18 @@
...
+    "migrate:undo:all": "sequelize-cli db:migrate:undo:all",
+    "migrate:all": "sequelize-cli db:migrate",
+    "seed:all": "sequelize-cli db:seed:all",
+    "seed:undo:all": "sequelize-cli db:seed:undo:all",
+    "seed": "npm run seed:undo:all && npm run seed:all"
```
Comment (fab-ryan) [inline package.json]: some as here

File: package.json (hunk 1 above)
Comment (fab-ryan) [reply package.json]: you didn't change anything

**Review by fab-ryan — APPROVED** [review]
great work

**Review by kirengaboris — APPROVED** [review]

**Review by ntarindwabujuli — APPROVED** [review]

**Review by istafrica-dennw — APPROVED** [review]

**Pull request merged**